glean-core/src/metrics/mod.rs

```diff
@@ -152,6 +153,32 @@ pub enum Metric {
     Object(String),
 }
 
+impl MallocSizeOf for Metric {
+    fn size_of(&self, ops: &mut malloc_size_of::MallocSizeOfOps) -> usize {
+        match self {
```

Comment:
Guess there's no way to say "sum `MallockSizeOf::size_of` with `ops` over all tuple members, regardless of enum variant", huh. Dang.

Reply:
nope